Formula/p/purr.rb

```diff
@@ -0,0 +1,30 @@
+class Purr < Formula
+  desc "Versatile zsh CLI tool for viewing and searching through Android logcat output"
+  homepage "https://github.com/google/purr"
+  url "https://github.com/google/purr/archive/refs/tags/2.0.0.tar.gz"
+  sha256 "f4e6c3dcdd386da7d3664fe6c947e9cfdb2e3362ca8c78e47d0445d94d471246"
+  license "Apache-2.0"
+  head "https://github.com/google/purr.git", branch: "main"
+
+  depends_on "make" => :build
+  depends_on "fzf"
+  depends_on "zsh"
+
+  def install
+    system "make", "purr"
+    bin.install "out/purr"
+  end
+
+  def caveats
+    <<~EOS
+      purr requires you to have adb executable in $PATH
+
+      adb is part of Android SDK command line tools. Get them at:
+        https://developer.android.com/studio#command-line-tools-only
+    EOS
+  end
+
+  test do
+    assert_match version.to_s, shell_output(bin/"purr -v")
```

Comment:
version check is not good enough

Reply:
Yes, I just thought that would be "better than nothing" for now. Fortunately author of `purr` (@alpiccioni) [has an idea](https://github.com/Homebrew/homebrew-core/pull/156118#issuecomment-1836369501) on how to make it work, and [suggestions](https://github.com/Homebrew/homebrew-core/pull/156118#issuecomment-1836374186) from @alebcay here are also very helpful. Thank you!

Reply:
we still need some test improvements on this new formula :)

Reply:
yeah definitely, I'll write a better test tomorrow (if time permits) or on Monday. Thank you for your support!